data/sql/updates/pending_db_world/rev_1588338801814673000.sql

```diff
@@ -0,0 +1,61 @@
+UPDATE `creature_template` SET `mindmg` = 179, `maxdmg` = 253, `DamageModifier` = 1.03 WHERE `entry` = 17357;
+UPDATE `creature_template` SET `mindmg` = 242, `maxdmg` = 341, `DamageModifier` = 1.03 WHERE `entry` = 20566;
+UPDATE `creature_template` SET `mindmg` = 823, `maxdmg` = 1164, `DamageModifier` = 1.03 WHERE `entry` = 17695;
+UPDATE `creature_template` SET `mindmg` = 1099, `maxdmg` = 1556, `DamageModifier` = 1.03 WHERE `entry` = 20580;
+UPDATE `creature_template` SET `mindmg` = 789, `maxdmg` = 1115, `minrangedmg` = 787, `maxrangedmg` = 1169, `DamageModifier` = 1.03 WHERE `entry` = 17427;
+UPDATE `creature_template` SET `mindmg` = 1056, `maxdmg` = 1493, `minrangedmg` = 1057, `maxrangedmg` = 1570, `DamageModifier` = 1.03 WHERE `entry` = 20579;
+UPDATE `creature_template` SET `mindmg` = 857, `maxdmg` = 1213, `DamageModifier` = 1.03 WHERE `entry` = 17461;
+UPDATE `creature_template` SET `mindmg` = 1099, `maxdmg` = 1556, `DamageModifier` = 1.03 WHERE `entry` = 20581;
+UPDATE `creature_template` SET `mindmg` = 857, `maxdmg` = 1213, `DamageModifier` = 1.03 WHERE `entry` = 17465;
+UPDATE `creature_template` SET `mindmg` = 1099, `maxdmg` = 1556, `DamageModifier` = 1.03 WHERE `entry` = 20583;
+UPDATE `creature_template` SET `mindmg` = 823, `maxdmg` = 1164, `DamageModifier` = 1.03 WHERE `entry` = 17464;
+UPDATE `creature_template` SET `mindmg` = 1056, `maxdmg` = 1493, `DamageModifier` = 1.03 WHERE `entry` = 20586;
+UPDATE `creature_template` SET `mindmg` = 857, `maxdmg` = 1213, `DamageModifier` = 1.03 WHERE `entry` = 17671;
+UPDATE `creature_template` SET `mindmg` = 1099, `maxdmg` = 1556, `DamageModifier` = 1.03 WHERE `entry` = 20584;
+UPDATE `creature_template` SET `mindmg` = 169, `maxdmg` = 239, `DamageModifier` = 1.03 WHERE `entry` = 17623;
+UPDATE `creature_template` SET `mindmg` = 176, `maxdmg` = 250, `DamageModifier` = 1.03 WHERE `entry` = 20575;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1081, `maxdmg` = 1531, `DamageModifier` = 1.01 WHERE `entry` = 16807;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1120, `maxdmg` = 1584, `DamageModifier` = 1.01 WHERE `entry` = 20568;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 918, `maxdmg` = 1302, `DamageModifier` = 1.01 WHERE `entry` = 16809;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1178, `maxdmg` = 1670, `DamageModifier` = 1.01 WHERE `entry` = 20596;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1129, `maxdmg` = 1601, `DamageModifier` = 1.01 WHERE `entry` = 16808;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1178, `maxdmg` = 1670, `DamageModifier` = 1.01 WHERE `entry` = 20597;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1254, `maxdmg` = 1778, `DamageModifier` = 1.01 WHERE `entry` = 20923;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1609, `maxdmg` = 2281, `DamageModifier` = 1.01 WHERE `entry` = 20993;
```

Comment:
Correct the flags assignment like the wiki describe https://github.com/azerothcore/wiki/blob/master/docs/Dealing-with-SQL-files.md#flags